frontend/test/metabase/scenarios/question/new.cy.spec.js

```diff
@@ -299,18 +299,11 @@ describe("scenarios > question > new", () => {
     });
 
     it("should correctly choose between 'Object Detail' and 'Table (metabase#13717)", () => {
-      // set ID to `No semantic type`
-      cy.request("PUT", `/api/field/${ORDERS.ID}`, {
-        semantic_type: null,
-      });
-      // set Quantity to `Entity Key`
```

Comment:
This is a test for composite PK scenario (> 1 PK columns), but making the "Quantity" a PK and removing the PK type for "ID" doesn't make sense in this case